proto/ai/traceable/agent/config/v1/config.proto

```diff
@@ -46,6 +46,9 @@ message BlockingConfig {
   // when `true`, blocking evaluation will be skipped for internal requests i.e. requests coming
   // from private IPs
   google.protobuf.BoolValue skip_internal_request = 7;
+
+  // Allows user to set a custom blocking status code value
+  google.protobuf.Int32Value blocking_status_code = 8;
```

Comment:
I think clearer to call it `response_status_code`.

Reply:
Can do it.